spec/test-suite.json

```diff
@@ -0,0 +1,71 @@
+{
+    "suite":
+    {
+        "zmanim":
+        [
+            {
+                "setup":
+                {
+                    "calculator":
+                    {
+                        "name": "SunTimes",
+                        "zmanimInformation":
+                        {
+                            "name": "The SunTimes calculator based on the USNO algorithm.",
+                            "url": "https://github.com/KosherJava/zmanim/blob/master/src/net/sourceforge/zmanim/util/SunTimesCalculator.java"
+                        },
+                        "information":
+                        {
+                            "name": "US Naval Observatory's Almanac for Computer algorithm",
+                            "url": "http://aa.usno.navy.mil/publications/docs/asa.php"
+                        }
+                    },
+                    "date": "2010-04-02",
+                    "location":
+                    {
+                        "name": "Lakewood, NJ",
+                        "lat": 40.09596,
+                        "long": -74.22213,
+                        "elevation": 0,
+                        "timeZone":
+                        {
+                            "offset": -5,
+                            "zoneInfoName": "America/New_York"
+                        }
+                    }
+                },
+                "tests":
+                [
+                    {
+                        "method": "sunset",
+                        "description": "Gets the exact sunset time",
+                        "extra":
+                        {
+                            "edgeCase": false,
```

Comment:
Not sure this is entirely necessary.

Reply:
This isn't necessary but having this kind of information in the test output can be nice. But might not be worth it.